apps/android/app/src/main/java/chat/simplex/app/views/chat/ChatView.kt

```diff
@@ -227,8 +227,13 @@ fun ChatView(chatId: String, chatModel: ChatModel, onComposed: () -> Unit) {
         }
       },
       acceptFeature = { contact, feature ->
+        val prefs = contact.mergedPreferences.toPreferences()
```

Comment:
you need to pass parameter here from the caller, not to decide on it here

Reply:
and also use contact.mergedPreferences.timedMessages.userPreference.preference.ttl